src/pages/settings/ResendValidationMessage.js

```diff
@@ -0,0 +1,74 @@
+import Str from 'expensify-common/lib/str';
```

Comment:
I don't think that we are reusing this component anywhere. Maybe we can just move this logic to `ResendValidationLinkPage`

Reply:
I just use `Str` to check the user logins by email or sms: 
`const isSMSLogin = Str.isSMSLogin(props.credentials.login);`

Reply:
I meant the `ResendValidationMessage.js` component. I'm not sure that we should make it its own separate component